packages/debug/src/browser/console/debug-console-session.ts

```diff
@@ -42,6 +43,8 @@ export class DebugConsoleSession extends ConsoleSession {
 
     protected readonly completionKinds = new Map<DebugProtocol.CompletionItemType | undefined, monaco.languages.CompletionItemKind>();
 
+    sessionManager: ConsoleSessionManager;
```

Comment:
You should be able to change this to `@inject(ConsoleSessionManager) protected readonly sessionManager: ConsoleSessionManager;` and then not change the interface for the `DebugConsoleSessionFactory` - the `ConsoleSessionManager` will be provided automatically by the call to `context.container.get(DebugConsoleSession)`.

Reply:
Good catch! That was an unnecessary change indeed, I fixed it, thank you!